scripts/create_devmapper.sh

```diff
@@ -6,6 +6,12 @@ pushd /var/lib/firecracker-containerd/snapshotter/devmapper > /dev/null
 DIR=/var/lib/firecracker-containerd/snapshotter/devmapper
 POOL=fc-dev-thinpool
 
+CONTAINERID=$(basename $(cat /proc/1/cpuset))
```

Comment:
what if I want to run this script on the bare host (no docker)?

Reply:
`basename $(cat /proc/1/cpuset)` returns "\\" when executed on bare host. When this script is executed on bare host, thinpool device named "fc-dev-thinpool" is created instead of "<container id>_thinpool".

Reply:
add a comment